tests/cfgs/default/result/custom_rules_ipv6.pcapng.out

```diff
@@ -21,17 +21,17 @@ Patricia risk IPv6:   0/0 (search/found)
 Patricia protocols:   0/0 (search/found)
 Patricia protocols IPv6: 9/5 (search/found)
 
-Unknown	1	318	1
+Unknown	1	1287	1
 CustomProtocolD	2	600	1
 CustomProtocolE	1	1287	1
-CustomProtocolF	1	1287	1
+CustomProtocolG	1	318	1
 CustomProtocolH	1	318	1
```

Comment:
???? Why these changes?
For example, the session `UDP [247f:855b:5e16:3caf:3f2c:4134:9592:661b]:36098 -> [21bc:b273:7f68:88d7:77a8:585:3990:927b]:50621` changed from `CustomProtocolF` to `CustomProtocolC`!!!!

Do you have any idea why (the issue, if real, seems totally unrelated to your change)?

Reply:
> ???? Why these changes? For example, the session `UDP [247f:855b:5e16:3caf:3f2c:4134:9592:661b]:36098 -> [21bc:b273:7f68:88d7:77a8:585:3990:927b]:50621` changed from `CustomProtocolF` to `CustomProtocolC`!!!!
> 
> Do you have any idea why (the issue, if real, seems totally unrelated to your change)?

I dunno,  maybe it's caused by protocol ID incrementation?

Reply:
Found the issue. I will try to push a fix asap